config/settings.js

```diff
@@ -27,11 +35,19 @@
  */
 
 window.spinnakerSettings = {
+  defaultProviders: ['aws', 'gce', 'azure', 'cf', 'kubernetes'],
```

Comment:
Should this list be constructed dynamically based on these services being enabled? This is just copied from deck's settings.js.